pkgs/development/python-modules/asonic/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "asonic";
+  version = "2.0.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-cw9Pk4DQBH48RA3d0C63k2V2hsiQqNx1bQI3r0nhki8=";
+  };
+
+  build-system = [ setuptools ];
+
+  pythonImportsCheck = [ "asonic" ];
+
```

Comment:
Please add a comment why the tests are not run, or enable tests using `nativeCheckInputs = [ pytestCheckHook ]`